src/app/functions/server/onerep.ts

```diff
@@ -161,6 +161,9 @@ export async function createProfile(
     updated_at: ISO8601DateString;
     url: string;
   } = await response.json();
+
+  logger.info("onerep_profile_created", { onerepProfileId: savedProfile.id });
```

Comment:
Is it fine for us to log the OneRep profile id in clear text or is this PII?

Reply:
We've been only logging IDs for reporting purposes (and not the actual data it refers to) which is personally identifiable, but not without credentials. I don't think we *need* this one, but in order to track some things like how long scans take we do need the scan ID. I'll remove it for now and follow up to make sure this is OK to do.